test/Microsoft.AspNet.Mvc.FunctionalTests/FiltersTest.cs

```diff
@@ -155,7 +155,7 @@ public class FiltersTest
                 "http://localhost/AuthorizeUser/Impossible");
 
             // Assert
-            Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
+            Assert.Equal(HttpStatusCode.Forbidden, response.StatusCode);
```

Comment:
Yeah this is tracking the case that fails https://github.com/aspnet/Security/issues/300